src/04_printing.py

```diff
@@ -13,5 +13,6 @@
 # x is 10, y is 2.25, z is "I like turtles!"
 
 # Use the 'format' string method to print the same thing
-
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print(f'x is {x}, y is {y}, z is "{z}"')
+# Finally, print the same thing using an f-string
+print("x is %s, y is %s, z is %s" % (x, y, z))
```

Comment:
be sure to really read through what is being asked of you. Be sure to re-visit printf operators